DOCUMENTATION.md

```diff
@@ -2377,6 +2378,50 @@ let webApp =
     ]
 ```
 
+### Request Limitation
+
+The type of requests that are allowed can be guard or limited. Requests with a certain type of `Accept` or `Content-Type` header can be checked for acceptable values and a user-friendly error message is send back to the consumer automatically when the conditions are not met.
+
+Giraffe offers several http handlers that can be part of your request processing pipeline.
+
+**Accept**
+Guards http request based on its `Accept` header:
+
+```fsharp
+// Only allow http requests with an `Accept` header equals `application/json`.
+let webApp =
+  GET >=> mustAcceptAny [ "application/json" ] >=> text "Hello World"
+
+// Http request with `Accept` = `application/json`      -> pass through
+// Http request without `Accept` = `application/json    -> error 406 http response: "cannot accept request because header 'Accept' hasn't got expected MIME type"
+```
+
+**Content-Type**
+Guards http request based on its `Content-Type` header:
+
+```fsharp
+// Only allow http request with a `Content-Type` header `equals `application/json`.
+let webApp =
+  GET >=> haveContentType "application/json" >=> text "Hello World"
+
+// Http request with    `Content-Type` = `application/json`   -> pass through
+// Http request without `Content-TYpe` = `application/json`   -> error 406 http response: "cannot accept request because expected to have request header 'Content-Type'"
```

Comment:
```suggestion
// Http request without `Content-Type` = `application/json`   -> error 406 http response: "cannot accept request because expected to have request header 'Content-Type'"
```